packages/suite/src/components/wallet/AccountsMenu/AccountItem.tsx

```diff
@@ -163,30 +170,54 @@ export const AccountItem = forwardRef((props: AccountItemProps, ref: React.Ref<H
                     <Row>
                         <AccountName data-test={`${dataTestKey}/label`}>{accountLabel}</AccountName>
                     </Row>
-                    <Row>
-                        <Balance>
-                            <CoinBalance value={account.formattedBalance} symbol={account.symbol} />
-                        </Balance>
-                        {(account.networkType === 'ethereum' ||
-                            account.networkType === 'cardano') &&
-                            !!account.tokens?.length && (
-                                <TokensCount
-                                    count={account.tokens.length}
-                                    onClick={handleClickOnTokens}
+                    {isBalanceShown && (
```

Comment:
Could use a ternary operator, `isBalanceShown ? (` etc.

Reply:
I try to avoid using ternary if both expressions are too long to be seen as a whole, I found it less readable.